File: hr_parent/hrm_common_model/src/main/java/com/hrm/domain/company/Company.java
package com.hrm.domain.company;

import lombok.*;
import org.hibernate.Hibernate;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.io.Serializable;
import java.util.Date;
import java.util.Objects;

/**
 * @author AsheOne
 * @date 2022/2/2
 */
@Entity
@Table(name = "co_company")
@Getter
@Setter
@ToString
@RequiredArgsConstructor
@AllArgsConstructor
@NoArgsConstructor
public class Company implements Serializable {

    private static final long serialVersionUID = 594829320797158219L;
    /**
     * ID
     */
    @Id
    private String id;
    /**
     * Company Name
     */
    private String name;
    /**
     * Enterprise Login ID
     */
    private String managerId;
    /**
     * Current Version
     */
    private String version;
    /**
     * Renewal Time
     */
    private Date renewalDate;
    /**
     * Expiration Date
     */
    private Date expirationDate;
    /**
     * Company Area
     */
    private String companyArea;
    /**
     * Company Address
     */
    private String companyAddress;
    /**
     * License Image ID
     */
    private String businessLicenseId;
    /**
     * Legal Representative
     */
    private String legalRepresentative;
    /**
     * Company Phone
     */
    private String companyPhone;
    /**
     * Email
     */
    private String mailbox;
    /**
     * Company Size
     */
    private String companySize;
    /**
     * Company Industry
     */
    private String industry;
    /**
     * Info Remarks
     */
    private String remarks;
    /**
     * Audit State
     */
    private String auditState;
    /**
     * Current State
     */
    private String state;
    /**
     * Current Balance
     */
    private Double balance;
    /**
     * Create Time
     */
    private Date createTime;

    @Override
    public boolean equals (Object o) {
        if (this == o) return true;
        if (o == null || Hibernate.getClass(this) != Hibernate.getClass(o)) return false;
        Company company = (Company) o;
        return id != null && Objects.equals(id, company.id);
    }

    @Override
    public int hashCode () {
        return getClass().hashCode();
    }
}

File: hr_web/src/mock/company.js
import Mock from 'mockjs'
import { param2Obj } from '@/utils'



const List = []
const count = 100

for (let i = 0; i < 3; i++) {
    let data = {
        id: "1"+i,
        name: "浼佷笟"+i,
        managerId: "string",
        version: "璇曠敤鐗坴1.0",
        renewalDate: "2018-01-01",
        expirationDate: "2019-01-01",
        companyArea: "string",
        companyAddress: "string",
        businessLicenseId: "string",
        legalRepresentative: "string",
        companyPhone: "13800138000",
        mailbox: "string",
        companySize: "string",
        industry: "string",
        remarks: "string",
        auditState: "string",
        state: "1",
        balance: "string",
        createTime: "string"
    }
  List.push(data)
}

export default {
  list: () => {
    return {
        code: 10000,
        success: true,
        message: "鏌ヨ鎴愬姛",
        data:List
    }
  },
  sassDetail:() => {
    return {
      code: 10000,
      success: true,
      message: "鏌ヨ鎴愬姛",
      data:{
        id: "10001",
        name: "娴嬭瘯浼佷笟",
        managerId: "string",
        version: "璇曠敤鐗坴1.0",
        renewalDate: "2018-01-01",
        expirationDate: "2019-01-01",
        companyArea: "string",
        companyAddress: "string",
        businessLicenseId: "string",
        legalRepresentative: "string",
        companyPhone: "13800138000",
        mailbox: "string",
        companySize: "string",
        industry: "string",
        remarks: "string",
        auditState: "string",
        state: "1",
        balance: "string",
        createTime: "string"
      }
    }
  }
}
